packages/grid/src/index.scss

```diff
@@ -48,7 +48,9 @@ $grid-gutter-width: 20px !default;
  */
 .is-debug {
   .row > [class*='col'] {
+    /* stylelint-disable-next-line color-function-notation */
```

Comment:
Ignore legacy notion to support current sass/scss `rgba` function.

https://sass-lang.com/documentation/modules#rgb